packages/components/src/FormatFile/FormatFile.tsx

```diff
@@ -32,46 +75,132 @@ export function FormatFile({ file, onDelete }: FormatFileProps) {
   }
 
   const style = imageSource ? { backgroundImage: `url(${imageSource})` } : {};
+  const imageBlockStyle = isComplete
+    ? styles.imageBlock
+    : classNames(styles.imageBlock, styles.imageBlockOverlay);
 
   return (
-    <div className={styles.formatFile}>
-      <div className={styles.imageBlock} style={style} data-testid="imageBlock">
+    <div
+      onMouseEnter={() => {
```

Comment:
By using mouse events it is no longer possible to get the delete button to render using keyboard focus

Reply:
I think onFocus and onBlur might solve this

Reply:
With just using onBlur and onFocus, we lose the delete button appearing on hover, which was the original intention. We will probably need both onFocus/onBlur as well as onMouseOver to get full functionality.

![hover-doesnt-work](https://user-images.githubusercontent.com/34727471/171906534-dbcadbf9-ceb7-43ac-99d2-f9e51bd99b36.gif)